views/js/previewer/plugins/button.tpl

```diff
@@ -0,0 +1,19 @@
+<li
+    data-control="{{control}}"
+    class="small btn-info action{{#if className}} {{className}}{{/if}}"
+    title="{{title}}"
+    role="button"
+    {{#each aria}}
+        aria-{{@key}}="{{this}}"
+    {{/each}}
+>
+    <a
```

Comment:
It'd be better to use `button` tag instead of `a` since the `href` is always `#`.